package.json

```diff
@@ -77,15 +77,6 @@
                         "armv7l",
                         "arm64"
                     ]
-                },
-                {
-                    "target": "pacman",
```

Comment:
I noticed you removed the Pacman build - how come?

Reply:
It's not the arch's pacman build afaik... so i removed it since i don't know what it was built for.

Reply:
i added the pacman build in the linux fix commit, but figured out soon after that it wasn't arch linux's pacman build so i deleted it.